op-node/flags/flags.go

```diff
@@ -372,6 +372,13 @@ var (
 		Value:    time.Second * 1,
 		Category: SequencerCategory,
 	}
+	BlockBuildingThresholdFlag = &cli.Uint64Flag{
+		Name:     "block-building.threshold",
+		Usage:    "Block building healthcheck threshold (default: 2 seconds)",
+		EnvVars:  prefixEnvVars("BLOCK_BUILDING_THRESHOLD"),
+		Value:    2,
+		Category: RollupCategory,
+	}
```

Comment:
Should we make the threshold in ms instead of seconds? That way it allows for more granularity.